File: data/java/GCD_ELEMENTS_GIVEN_RANGE_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<Integer> param0 = new ArrayList<>();
    param0.add(57);
    param0.add(22);
    param0.add(17);
    param0.add(74);
    param0.add(93);
    param0.add(56);
    param0.add(5);
    param0.add(5);
    param0.add(9);
    param0.add(98);
    List<Integer> param1 = new ArrayList<>();
    param1.add(57);
    param1.add(22);
    param1.add(17);
    param1.add(74);
    param1.add(22);
    param1.add(54);
    param1.add(33);
    param1.add(68);
    param1.add(75);
    param1.add(21);
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
	        if(f_filled(param0.get(i),param1.get(i)) == f_gold(param0.get(i),param1.get(i)))
	        {
                builder.append("GCD_ELEMENTS_GIVEN_RANGE," + args[1] + "," + i + ",SUCCESS\n");
            } else {
                builder.append("GCD_ELEMENTS_GIVEN_RANGE," + args[1] + "," + i + ",FAILURE\n");
            }
        } catch (Exception e) {
            builder.append("GCD_ELEMENTS_GIVEN_RANGE," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}

File: data/python/GCD_ELEMENTS_GIVEN_RANGE_MAIN.py
if __name__ == '__main__':
    param = [
    (57,57,),
    (22,22,),
    (17,17,),
    (74,74,),
    (93,22,),
    (56,54,),
    (5,33,),
    (5,68,),
    (9,75,),
    (98,21,)
        ]
    n_success = 0
    print("class,mutant,test_index,result")
    for i, parameters_set in enumerate(param):
        try:
            print("GCD_ELEMENTS_GIVEN_RANGE," + sys.argv[1] + "," + str(i) + "," + str(f_gold(*parameters_set)))
        except:
            print("GCD_ELEMENTS_GIVEN_RANGE," + sys.argv[1] + "," + str(i) + ",EXCEPTION")
